components/applications-service/pkg/storage/postgres/db.go

```diff
@@ -11,19 +11,19 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-// postgres is a wrapping struct that will hold the database mapping object
+// Postgres is a wrapping struct that will hold the database mapping object
```

Comment:
Had to make this public so I could embed it in the `Ingester`. But also it's kind of a pain to return private types from public functions. That said, we could make the members of the struct private if desired.